dcrjson/dcrwalletextcmds.go

```diff
@@ -248,6 +248,27 @@ func NewImportScriptCmd(hex string, rescan *bool, scanFrom *int) *ImportScriptCm
 	return &ImportScriptCmd{hex, rescan, scanFrom}
 }
 
+// CreateVotingAccountCmd is a type for handling custom marshaling and
+// unmarshaling of createvotingaccount JSON wallet extension commands.
+type CreateVotingAccountCmd struct {
+	Name       string
+	PubKey     string
+	ChildIndex *uint32 `jsonrpcdefault:"0"`
+}
+
+// NewCreateVotingAccountCmd creates a new CreateVotingAccountCmd.
+func NewCreateVotingAccountCmd(name, pubKey string, childIndex *uint32) *CreateVotingAccountCmd {
+	return &CreateVotingAccountCmd{name, pubKey, childIndex}
+}
+
```

Comment:
Missing the function comment for godoc and consistency.  It'll fail linters as well for that reason.